src/Controls/tests/Xaml.UnitTests/Issues/Maui25172.xaml.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Controls.Core.UnitTests;
+using Microsoft.Maui.Dispatching;
+using Microsoft.Maui.UnitTests;
+using NUnit.Framework;
+
+namespace Microsoft.Maui.Controls.Xaml.UnitTests;
+
+[XamlCompilation(XamlCompilationOptions.Skip)]
```

Comment:
remove this, and also test that the instantiated type inherits from the right type

Reply:
fixed